others/sub_sequences.cpp

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2019 by Contributors */
```

Comment:
please add documentation in [doxygen style](https://www.doxygen.nl/manual/docblocks.html) - you can also refer to #807 for an example